src/lib.rs

```diff
@@ -538,6 +586,92 @@ impl PublicKey {
         })
     }
 
+     // Partially Blind a message to be signed with public metadata
```

Comment:
As this will be part of the public API for RSAPBSSA, this should have a docstring, too.

Reply:
done